app2/src/main/java/com/example/recipebox/ui/theme/Type.kt

```diff
@@ -9,8 +9,8 @@ import androidx.compose.ui.unit.sp
 // Set of Material typography styles to start with
 val Typography = Typography(
     body1 = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.Normal,
+        fontFamily = FontFamily.SansSerif,
```

Comment:
Your design doc used sans-serif font, so I would recommend modifying the type class to reflect that.